### PR TITLE
Search Results: Result count per type

### LINKED ISSUE #1120: Search Results: Result count per type
## Feature Description

Related: #772, #773

When searching and filtering results on the dashboard, we need to show the total number of results, as well as the number of results per status (e.g. 10 totals, 5 drafts, 5 published). WordPress currently only reports the totals irregardless of status (i.e. 10 totals).

The search view also has a count for found templates. Given that templates are likely to be their own entity / post type, this probably requires some more tinkering, which we can do once we got to it.

[**Slack discussion**](https://xwp.slack.com/archives/CCCAUAH6F/p1586365385012400)
[**Example on Figma**](https://www.figma.com/file/hUTw97921an0e3uU28JoDn/Dashboard?node-id=1936%3A82064)

Screenshot:

Screenshot 2020-04-08 at 21 45 14

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Fix for #1120 count being wrong.

Fixes #1120

### CODE REVIEW

File: includes/REST_API/Stories_Controller.php [hunk 1]
```diff
@@ -356,15 +356,25 @@ public function get_items( $request ) {
...
+			'draft'   => 'draft',
+		];
+
+		$statuses_count = [];
+
+		// Strip down query for speed.
+		$query_args['fields']                 = 'ids';
+		$query_args['posts_per_page']         = 1;
+		$query_args['update_post_meta_cache'] = false;
+		$query_args['update_post_meta_cache'] = false;
```
Comment (swissspidy) [inline includes/REST_API/Stories_Controller.php]: The line is duplicated

```suggestion
```

Comment (spacedmonkey) [reply includes/REST_API/Stories_Controller.php]:
```suggestion
		$query_args['update_post_term_cache'] = false;
```

**Comment by swissspidy** [comment]
@BrittanyIRL Could you quickly test this by merging this branch into your `feature/1783-add-counts-by-story-status` branch locally?

**Review by swissspidy — APPROVED** [review]

**Comment by BrittanyIRL** [comment]
> @BrittanyIRL Could you quickly test this by merging this branch into your `feature/1783-add-counts-by-story-status` branch locally?

Looks good! thanks for the quick fix.

counts right
search counts right

**Pull request merged**